watchtower-plugin/tests/test.py

```diff
@@ -123,7 +120,7 @@ def test_auto_retry_watchtower(node_factory, bitcoind, teosd):
             {},
             {
                 "plugin": WT_PLUGIN,
-                "allow_broken_log": True,
+                "broken_log": r'plugin-watchtower-client: Data was send to an idle retier. This should have never happened. Please report!.*',
```

Comment:
This made me realize that there was a typo on this logline. Would you care changing it here and on the plugin side?

retier -> retrier